packages/flutter_localizations/lib/src/l10n/cupertino_es.arb

```diff
@@ -18,10 +18,10 @@
   "cutButtonLabel": "Cortar",
   "copyButtonLabel": "Copiar",
   "pasteButtonLabel": "Pegar",
-  "selectAllButtonLabel": "Seleccionar todo",
+  "selectAllButtonLabel": "Seleccionar todos",
```

Comment:
This is incorrect, see https://github.com/flutter/flutter/issues/127871
Same for _es_419.arb etc.

Reply:
Submitted translation internally.